be/src/exec/pipeline/pipeline_builder.cpp

```diff
@@ -14,7 +14,8 @@ OpFactories PipelineBuilderContext::maybe_interpolate_local_broadcast_exchange(R
     }
 
     auto pseudo_plan_node_id = next_pseudo_plan_node_id();
-    auto mem_mgr = std::make_shared<LocalExchangeMemoryManager>(state->chunk_size() * num_receivers * num_receivers);
+    auto mem_mgr = std::make_shared<LocalExchangeMemoryManager>(state->chunk_size() * num_receivers * num_receivers *
```

Comment:
why double multiply num_receivers？

Reply:
That's wrong, the correct formula should be `chunk_size * num_receiver * 8`.